### ISSUE TITLE
Python oauth 2 not working

### ISSUE THREAD

**Issue by pedrovelho** [HubSpot/sample-apps-oauth#23]
Hi,

I am following this sample to make a python API call to hubspot, however hubspot seems to reach an error every time I try to oauth2:

**This install link is no longer valid. Please visit [HubSpot's App Marketplace](https://ecosystem.hubspot.com/marketplace/apps) to install this app.**

The callback is set to `http://localhost:5000/oauth/callback` which seems to be the case for the python version. But the callback url is never called by the browser.

**Comment by pedrovelho** [comment]
So I figured out the problem. We need to use the link to install the app for the user first. Like this the user authorizes the app on the requested scope. After that the python application example worked. The link for installing the app is the one on the app configuration **Basic Info** -> **Auth**, copy and paste the install URL in your browser and select the desired user from there.

I also had to change the requests a bit since some dependencies break the newest version of Flask.

`requirements.txt`
```
Flask==2.0.1
hubspot-api-client==3.4.2
python-dotenv==0.15.0
jinja2<3.1.0
markupsafe<2.1.0
```
